share/doc/revng/references/mime-types.md

```diff
@@ -21,7 +21,7 @@ USAGE: revng-artifact [options] <artifact> <binary>
   make-segment-ref            - application/x.llvm.bc+zstd
   decompile                   - text/x.c+ptml+tar+gz
   decompile-to-single-file    - text/x.c+ptml
-  emit-recompilable-archive   - application/x-tar
+  emit-recompilable-archive   - application/x.recompilable-archive
```

Comment:
Mmmh, I'm not super happy to drop the MIME of tar.

Is this compressed? Below we do `+tar+gz`.

Reply:
Would `application/x.recompilable-archive+tar+gz` work?